recipes/triangle/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "triangle" %}
+{% set version = "20200424" %}
+{% set prefix = "py" %}
+
+package:
+  name: {{ prefix|lower }}-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fc207641f8f39986f7d2bee1b91688a588cd235d2e67777422f94e61fece27e9
+
+build:
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
```

Comment:
You should remove these comments before this is merged

Reply:
removed